actions/api/algolia.py

```diff
@@ -20,5 +35,6 @@ def get_algolia_link(self, hits: List, index: int):
         return doc_link
 
     def search(self, search_string: Text):
-        res = self.index.search(search_string)
+        search_text = preprocess_search_text(search_string)
+        res = self.index.search(search_text)
```

Comment:
same here, not sure if this should be in the class or in the `run` method of the action